codecs/avif/Makefile

```diff
@@ -1,7 +1,9 @@
-CODEC_URL = https://github.com/AOMediaCodec/libavif/archive/31d7c6d1e32cf467ac24fb8c7a76c4902a4c00db.tar.gz
+# Taken from https://source.corp.google.com/piper///depot/google3/third_party/libavif/METADATA
+CODEC_URL = https://github.com/AOMediaCodec/libavif/archive/v0.9.0.tar.gz
 CODEC_PACKAGE = node_modules/libavif.tar.gz
 
-LIBAOM_URL = https://aomedia.googlesource.com/aom/+archive/v2.0.0.tar.gz
+# Taken from https://source.corp.google.com/piper///depot/google3/third_party/libaom/METADATA
+LIBAOM_URL = https://aomedia.googlesource.com/aom/+archive/4a8e276e3e0ef3c76083f3975d5caa85bc9593ce.tar.gz
```

Comment:
Why? If you just want to update to the latest, https://aomedia.googlesource.com/aom shows 2.0.2. I don't think we care what internal third_party stores (or should link to it as a reference on Github).

Reply:
From Wan-Teh Chang:

> Also, we (Jingning in particular) recently improved the libaom encoder significantly to speed up still image encoding and reduce heap memory consumption. I suggest you use a recent stable snapshot of the libaom master branch. A good way is to use the snapshot checked into google3/third_party/libaom, which is roughly two weeks behind the tip of the master branch. You can find the git revision in https://source.corp.google.com/piper///depot/google3/third_party/libaom/METADATA

Reply:
Yeah, Jake's explanation above makes sense to me.